src/components/UserManagement/UserTableFooter.jsx

```diff
@@ -91,8 +91,11 @@ const PageSummaryLabel = React.memo(props => {
       className="table-summary col-md-4 col-sm-4 col-xs-4 ember-view"
       style={{ marginBottom: '25px' }}
     >
-      {`Show ${(props.selectedPage - 1) * props.pageSize + 1} -
-    ${props.selectedPage * props.pageSize} of ${props.datacount}`}
+      {`Show `}
```

Comment:
```suggestion
     {`Showing `}
      {props.datacount <= 10 && `${props.datacount}`}
      {props.datacount > 10 && props.datacount > props.selectedPage * props.pageSize
        && `${(props.selectedPage - 1) * props.pageSize + 1} -
    ${props.selectedPage * props.pageSize}`}
      {` of ${props.datacount}`}
```

Reply:
My suggestion on how to handle message depending on the number of users. Also I agree with a previous commenter that 'Showing' instead of 'Show' seems more correct.

Reply:
thank you